package.json

```diff
@@ -97,7 +97,7 @@
     "file-loader": "^0.9.0",
     "fs-extra": "^0.30.0",
     "hapi": "^13.5.0",
-    "history": "^2.0.0",
+    "history": "^3.0.0",
```

Comment:
We can't make this upgrade til react-router makes it to v3.  That's the reason for the build error.